test/box-luatest/box_cfg_env_test.lua

```diff
@@ -166,11 +166,7 @@ g.test_uri_list = function(g)
 
     local opts = {nojson = true, stderr = true}
     local res = justrun.tarantool(dir, {}, {'main.lua'}, opts)
-    t.assert_equals(res, {
-        exit_code = 0,
-        stdout = '',
-        stderr = '',
-    })
+    t.assert_equals(res.exit_code, 0, {res.stdout, res.stderr})
```

Comment:
Let's just set the logging level to "warning" in the script.

Reply:
This was the original solution. But after [discussion](https://github.com/tarantool/tarantool/pull/9996#discussion_r1602049854) with @Totktonada we decided to go this way.